talk/owt/sdk/p2p/p2pclient.cc

```diff
@@ -215,7 +215,8 @@ void P2PClient::OnSignalingMessage(const std::string& message,
       RTC_LOG(LS_WARNING) << "Non-existed chat cannot be stopped.";
       return;
     }
-  } else if (message.find("\"type\":\"offer\"") != std::string::npos) {
```

Comment:
Please check whether which one follows Chromium style.

Reply:
Not sure why this gets updated. Corrected to use chromium style.